flac.md

```diff
@@ -488,30 +488,59 @@ Value | Description
 
 The size of the samples stored in the subframe is the subframe sample size reduced by k bits. Decoded samples must be shifted left by k bits.
 
-## SUBFRAME_CONSTANT
-Data      | Description
-:---------|:-----------
-`u(n)`    | Unencoded constant value of the subblock, n = frame's bits-per-sample.
+## Constant subframe
```

Comment:
I would keep the paragraph titles as is there, as they are not modified in other parts. Modifying paragraph titles could be in a dedicated PR.